src/Cohesity.Powershell/Cmdlets/ProtectionJob/RemoveCohesityProtectionJob.cs

```diff
@@ -96,21 +96,28 @@ protected override void BeginProcessing()
         /// </summary>
         protected override void ProcessRecord()
         {
+            bool deleteSnapshots = true;
+            string deleteSnapshotsCautionMessage;
+
+            if (KeepSnapshots.IsPresent)
+            {
+                deleteSnapshotsCautionMessage = "This action keeps the snapshots generated by this protection job";
```

Comment:
Check c# line indent .. one for below too.

Reply:
I will change that but as a check do we use any lint tools for c#

Reply:
We should start one perhaps for a team for python, golang and c#